atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/EteSetup.java

```diff
@@ -116,6 +116,7 @@ public static RuleChain setup(
                 .machine(machine)
                 .saveLogsTo(LogDirectory.circleAwareLogDirectory(logDirectory))
                 .shutdownStrategy(ShutdownStrategy.AGGRESSIVE_WITH_NETWORK_CLEANUP)
+                .nativeServiceHealthCheckTimeout(org.joda.time.Duration.standardMinutes(5))
                 .build();
```

Comment:
Would it make sense to extract out the 5min duration to keep this and `CREATE_TRANSACTION_MANAGER_MAX_WAIT_TIME_SECS` in sync?

Reply:
Yeah, that seems fair.